src/conditionalLogicService/conditionallyShowElement.ts

```diff
@@ -0,0 +1,146 @@
+import { FormTypes, ConditionTypes } from '@oneblink/types'
+import { evaluateConditionalPredicate } from './evaluationService'
+
+export type FormElementsCtrl = {
+  model: Record<string, unknown>
+  flattenedElements: import('@oneblink/types').FormTypes.FormElement[]
```

Comment:
is there any reason why this cant be `FormTypes.FormElement[]`, you're already importing the package

Reply:
ah yeah, it can be.  That's how it is in apps-react.